src/components/Login/LoginOTPCard.tsx

```diff
@@ -120,20 +122,16 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
           description: e.message
         });
       } else if (e instanceof LoginLockedError) {
-        showAlert({
-          ...disabledAccessAlertProps,
-          description: e.message,
-          onOk: () => resetStage()
-        });
+        showAlert({ ...e.alertProps, onOk: () => resetStage() });
+      } else if (e instanceof OTPWrongError || e instanceof OTPExpiredError) {
+        showAlert(e.alertProps);
+      } else if (e instanceof AuthInvalidError) {
+        showAlert({ ...e.alertProps, onOk: () => resetStage() });
       } else if (e instanceof LoginError) {
-        showAlert({
-          ...invalidInputAlertProps,
-          description: e.message
-        });
+        showAlert(e.alertProps);
```

Comment:
actually thinking about this, for `LoginError`, whats the reason not to `resetStage`? in this case the user will be stuck at OTP screen. 
Also, maybe we can combine the types of errors in a single condition as well `(e instanceof LoginLockedError || e instanceof AuthInvalidError)

Reply:
i think in this case the `LoginError` is usually because of network error. `resetStage` should only be because of errors in the qr code or when the user is locked out. what cases did u have in mind when you asked this question?

Reply:
i was testing the use of a spoofed backend endpoint, the issue was the error message told me to try again, but in the end i couldn't because i was stuck in the otp card, need to kill the app.

Reply:
hmm that makes sense too...spoofed backend -> throws network error -> stuck in otp card. ok will change it to `resetStage` as well haha